lib/media_items/index.js

```diff
@@ -25,6 +25,23 @@ class MediaItems {
     })
   }
 
+  async uploadMultiple(albumId, files, directoryPath, description) {
```

Comment:
Wdyt about accepting an array of files with descriptions so that each file can have a different description?

Reply:
I could include that tonight.

Reply:
done